packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -468,9 +457,9 @@ export class FileStorage implements DStore {
       );
     }
 
-    this._addBacklinks({ notesWithLinks, allNotes });
+    this._addBacklinks({ notesWithLinks, allNotes: _.values(this.notes) });
```

Comment:
can we cache this value (aka instantiate separately) since we deconstruct this 2x (first time in line 450)?

Reply:
Good point. fixed